### ISSUE TITLE
bug: `onClick` override `href` attribute behavior when specified

### ISSUE THREAD

**Issue by ChaoticTempest** [NearSocial/VM#19]
Currently, if one were to specify a `onClick` handler inside an `a` tag that also has an `href` attribute, the `onClick` would override the link redirection behavior. It appears for `a` tags, `event.preventDefault()` is being used by default, and we should explicitly make the developers call this themselves. Also, one other thing to note, no matter where you put the `onClick`, in a parent tag, or in a child tag, the `onClick` will override any link redirection.

This might go into the territory of breaking changes for all the widgets out there, but more than likely not since this is just constrained to an `a` tag.

**Comment by marcinbodnar** [comment]
@ChaoticTempest Could you please describe it more precisely and provide an example?

**Comment by ChaoticTempest** [comment]
For example, the following widget when you click on it will not send you to the profile page, but only logs the event:
```js
const profileUrl = `/#/calebjacob.near/widget/ProfilePage?accountId=chaotictempest.near`;

return (
  <a
    href={profileUrl}
    onClick={(event) => {
      console.log("event", event);
    }}
  >
    Click
  </a>
);
```
I believe this is due an `event.preventDefault()` being inserted and used somewhere inside the VM which prevents navigating to the new page.

**Comment by mpeterdev** [comment]
I think this is the culprit

https://github.com/NearSocial/VM/blob/9d7ecff7bf50a114a59306b79f7d5a09706f06fe/src/lib/vm/vm.js#L1354

**Comment by calebjacob** [comment]
If anyone else runs into this issue, a creative workaround is to use the `onPointerUp` event (which fires for a mouse and touch):

```jsx
<a href="/" onPointerUp={() => console.log("asdf")}>Click</a>
```

@ChaoticTempest I double checked, and `onMouseDown` doesn't actually seem to fire on mobile after all, but `onPointerUp` should be just what you need.

**Comment by evgenykuzyakov** [comment]
The original issue was addressed in `2.5.0`.
